ctgan/data_transformer.py

```diff
@@ -46,7 +46,7 @@ def _fit_continuous(self, data):
                 A ``ColumnTransformInfo`` object.
         """
         column_name = data.columns[0]
-        gm = ClusterBasedNormalizer(model_missing_values=True, max_clusters=min(len(data), 10))
+        gm = ClusterBasedNormalizer(max_clusters=min(len(data), 10))
```

Comment:
to keep the behavior the same I think we have to change it to `missing_value_generation='from_column'`. The default is to fill with random nans